cfgov/v1/jinja2/v1/includes/organisms/secondary-navigation-fig.html

```diff
@@ -43,9 +43,11 @@ <h3>Table of contents</h3>
     {% from 'v1/includes/organisms/expandable.html' import expandable with context %}
     {% call() expandable(sec_nav_settings) %}
         {% import 'v1/includes/molecules/nav-link.html' as nav_link %}
-        <p class="u-hide-on-desktop">
-            Effective {{ effective_start_date }} to {{ effective_end_date }}
-        </p>
+        {% if version_status %}
+            <p class="u-hide-on-desktop">
```

Comment:
Hmmmm just noticing that we have `u-visually-hidden-on-mobile`, but this one is `u-hide-on-desktop`. We should probably reconcile those names some time.